eng/Version.Details.xml

```diff
@@ -35,41 +35,41 @@
     <Dependency Name="Microsoft.WindowsAppSDK" Version="0.0.1">
       <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal</Uri>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Runtime.win-x64" Version="6.0.3" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+    <Dependency Name="Microsoft.AspNetCore.App.Runtime.win-x64" Version="6.0.4" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
```

Comment:
Do you want to drop the `CoherentParentDependency` here? If you made these changes manually, then future Maestro bumps would undo this.

Related to the comment here: https://github.com/dotnet/maui/pull/5741#issuecomment-1097157880

Reply:
Thanks, confirming whether this is the path we want to take in terms of managing the dependencies on our end (ASP.NET core team). In the meantime, created https://github.com/dotnet/maui/pull/6014 so the CI can kick off.